packages/@glimmer/runtime/lib/vm/append.ts

```diff
@@ -35,16 +40,20 @@ export default class VM implements PublicVM {
   public listBlockStack = new Stack<ListBlockOpcode>();
   public frame = new FrameStack();
   public constants: Constants;
+  private notDone = { done: false, value: null };
```

Comment:
The escape analysis in Chrome is much improved with turbofan, and this will troll it.  Please use an object literal as the iterator result should not escape the call frame of the caller.